### PR TITLE
Rename ShardId to ShardOrd and shard_id to shard_ord

### PR DESCRIPTION
**Pull request by mzhangmzz**
As proposed in https://github.com/near/NEPs/pull/241

To avoid breaking the API, the shard_id fields in view structs are not changed

### CODE REVIEW

File: chain/indexer/src/streamer/types.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ pub struct IndexerExecutionOutcomeWithReceipt {
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct IndexerShard {
-    pub shard_id: types::ShardId,
+    pub shard_ord: types::ShardOrd,
```
Comment (bowenwang1996) [inline chain/indexer/src/streamer/types.rs:42]: @khorolets @frol would the change here break anything? Looks like the struct derives `Serialize`

File: core/primitives/src/views.rs [hunk 2]
```diff
@@ -648,7 +648,7 @@ pub struct ChunkHeaderView {
     pub encoded_length: u64,
     pub height_created: BlockHeight,
     pub height_included: BlockHeight,
-    pub shard_id: ShardId,
+    pub shard_ord: ShardOrd,
```
Comment (bowenwang1996) [inline core/primitives/src/views.rs:651]: Looks like you removed the old field directly here, which is not desirable.

**Review by bowenwang1996 — COMMENTED** [review]
@frol please review the changes related to rpc to make sure that we don't break any API

File: core/primitives/src/views.rs (hunk 2 above)
Comment (mzhangmzz) [reply core/primitives/src/views.rs:651]: hmm this changeset looks like the changeset I committed yesterday, not sure why, I thought I deleted that one. Let me try to re-submit

**Comment by mzhangmzz** [comment]
created a new one

**Pull request closed without merging**